README.md

```diff
@@ -1,61 +1,66 @@
 [![Build Status](https://travis-ci.org/daostack/daostack.svg?branch=master)](https://travis-ci.org/daostack/daostack)
 [![NPM Package](https://img.shields.io/npm/v/daostack-arc.svg?style=flat-square)](https://www.npmjs.org/package/daostack-arc)
+# DAOstack Arc
 
+Arc is the lower layer of the DAOStack. It consists of several smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
```

Comment:
`... layer of the DAOStack platform`?

Reply:
`the lower layer of the DAO stack`, maybe I should separate it with a space.